test/playbook-include/playbook_post.yml

```diff
@@ -1,8 +0,0 @@
----
-- hosts: localhost
-  tasks:
-    - command: echo "no name"  # noqa 502
-  post_tasks:
```

Comment:
no longer importing as post_tasks, hope this does not lose coverage

Reply:
Yeah. This is what I wanted.